src/backend/db/categories.js

```diff
@@ -5,23 +5,208 @@ import { v4 as uuid } from "uuid";
  * You can add category of your wish with different attributes
  * */
 
+// export const categories = [
+//   {
+//     _id: uuid(),
+//     categoryName: "Computer Programming",
+//     description:
+//       "Computer programming is the process of designing and building an executable computer program to accomplish a specific computing",
+//   },
+//   {
+//     _id: uuid(),
+//     categoryName: "Frontend Development",
+//     description:
+//       "Front-end web development, also known as client-side development is the practice of producing HTML, CSS and JavaScript for a website or Web Application",
+//   },
+//   {
+//     _id: uuid(),
+//     categoryName: "Backend Development",
+//     description:
+//       "Backend Development is also known as server-side development. It is everything that the users don't see and contains behind-the-scenes activities that occur when performing any action on a website.",
+//   },
+// ];
+
```

Comment:
Same, if not needed, can remove commented code.

Reply:
ok!